app/models/energy/simulationparameter.py

```diff
@@ -0,0 +1,180 @@
+"""Simulation Parameter Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError
+from typing import List, Union
+from enum import Enum
+from app.models.common.datetime import Date
+
+
+class ReportingFrequency(str, Enum):
+    detailed = 'Detailed'
+    timestep = 'Timestep'
+    hourly = 'Hourly'
+    daily = 'Daily'
+    monthly = 'Monthly'
+    run_period = 'RumPeriod'
+    environment = 'Environment'
+    annual = 'Annual'
```

Comment:
For now, I think we should only support the following options since the other ones can be represented with just these five:
```
    timestep = 'Timestep'
    hourly = 'Hourly'
    daily = 'Daily'
    monthly = 'Monthly'
    annual = 'Annual'
```
Also, I would love to run a simulation for a `RumPeriod`. It sounds intoxicating 😄 !